src/main/kotlin/no/nav/syfo/BootstrapApplication.kt

```diff
@@ -218,6 +227,12 @@ fun Application.serverModule(
         }
     }
 
+    runningInFSSCluster {
+        log.info("Migrating notifications")
+        val migrationStatus = migrationService.migrateUnsentVarsler()
```

Comment:
Er det slik at alle planlagte varsler blir migrert hver gang appen starter?

Reply:
Har du noe peiling på hvor lang tid migreringen vil ta? Hvis det tar veldig lang, tid, kan det forårsake noe problemer med oppstart av podden?

Reply:
Det er bare fremtidige planlagte varsler som vil migreres fra FSS. Det er snakk om underkant av 7600
varsler som skal migreres (hvis vi prodsatte i dag), og burde ikke ta så altfor lang tid. Kan for sikkerhets øke
timeouten på isReady endepunktet. Kom også på at vi må gjøre leader election for å hindre at alle podene
sender samtidig (bare 1 pod skal sende til topicen).